components/service/fretboard/src/main/java/mozilla/components/service/fretboard/source/kinto/KintoExperimentSource.kt

```diff
@@ -22,16 +23,24 @@ class KintoExperimentSource(
     val baseUrl: String,
     val bucketName: String,
     val collectionName: String,
-    private val client: HttpClient = HttpURLConnectionHttpClient()
+    client: HttpClient = HttpURLConnectionHttpClient()
 ) : ExperimentSource {
+    var validateSignature = true
```

Comment:
Is this something we could pass in via the constructor?

Reply:
Yes, it could be passed via the constructor, right. I'm going to change it